docs/inventory.md

```diff
@@ -151,10 +151,15 @@ monitoring:
 ## Mac Minis
 
 owner: the NixOS Foundation
-access: Dan, Eelco, Rob, Graham
-role: build machines
+access: Cole-h
+role: build machines at hetzner online
```

Comment:
These are not hetzner machines, but machines in Graham's basement :sweat_smile: 

The hetzner macs are ??? (Graham can't login anymore, maybe Eelco or Domen have access)

Reply:
Please check again.